src/components/appRouter.js

```diff
@@ -33,56 +34,22 @@ class AppRouter {
     startPages = ['home', 'login', 'selectserver'];
 
     constructor() {
-        // WebKit fires a popstate event on document load
```

Comment:
I don't _think_ we should hit this issue now since we are just listening to the `history` library events, but it should probably be tested.

Reply:
I checked the state of `isBack` on webOS 1.2 (closest to Tizen 2.3) and it seems to be valid on the first load - `false` (`action` is `undefined`).

For reference, the old `popstate` approach has `isBack: true` without `!loaded`.

We need Tizen 2.x to be sure, but either way it should be fixed in `history`.